selene-lib/tests/lints/roblox_incorrect_roact_usage/roblox_incorrect_roact_usage.stderr

```diff
@@ -16,3 +16,15 @@ error[roblox_incorrect_roact_usage]: `ThisDoesntExist` is not a valid class
 13 │ e("ThisDoesntExist", {})
    │   ^^^^^^^^^^^^^^^^^
 
+error[roblox_incorrect_roact_usage]: `Name` will never be applied for `Instance`
```

Comment:
You're formatting for a special class name, but it's only showing up as "Instance" here?

Personally, I think we don't need to specify the class name itself, I think just "native Roblox classes" is good enough, and makes it unambiguous as to what the problem is.